default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, dmd, nasm, llvmPackages, dflags ? null}:
+stdenv.mkDerivation
+{   name="sdc";
+    src= lib.cleanSource ./.;
+    nativeBuildInputs = [dmd llvmPackages.bintools];
```

Comment:
Isn't make, gcc and friends required here?

Reply:
No, those are part of the standard environment.

It's good thing you asked me to test again. I had made a mistake: the `buildInputs` should have `libllvm`, not `bintools`. It might be that I had a fit version of LDC installed or something and the linker found it's library when I last tested.

Anyway, now it compiles in both the unstable and 21.05 Nix channels.

Reply:
ping @deadalnix

Reply:
Sorry i kinda dropped the ball here. Merged.